acinclude/os-deps.m4

```diff
@@ -736,33 +732,28 @@ void squid_getprotobynumber(void) {
 }
   ])
   AC_MSG_CHECKING([for winsock library])
-  case "$ac_cv_search_squid_getprotobynumber" in
-    "no")
-      AC_MSG_RESULT([winsock library not found])
-      ;;
-    "none required")
-      AC_MSG_RESULT([winsock library already in LIBS])
-      ;;
-    "-lws2_32")
+  AS_CASE(["$ac_cv_search_squid_getprotobynumber"],
+    ["no"],[AC_MSG_RESULT([winsock library not found])],
```

Comment:
Quotes around case labels is an uncommon pattern, it may be the first time i see it. This is a pretty untested part of the code, I'm not even sure it works.
I'd leave as is because it's not worse than what we have now, unless you have the time to explore

Reply:
I'm not sure why it existed here TBH, so I left it as the fix is currently untestable.

Reply:
I'm setting up a Windows VM, hopefully this will change

Reply:
Resolving this discussion as wont change for now. It can be done later when testing is available.